docs/modules/policies/pages/conditions.adoc

```diff
@@ -29,6 +30,17 @@ Within a condition block you have access to several special variables:
 
 Every condition expression must evaluate to a boolean true/false value. You can combine complex expressions together in a single condition block by using the `all`, `any`, or `none` operators.
 
+.Quotes in expressions
+****
+Because single and double quotes have special meanings in YAML, if your expression contains quotation marks, use the YAML literal block syntax to avoid YAML parsing errors.
+
+[source,yaml]
+----
+expr: |
```

Comment:
We use the folded block style elsewhere I think:

```suggestion
Single and double quotes have special meanings in YAML. If your expression contains quotes, use the YAML block syntax to avoid parsing errors.

[source,yaml]
----
expr: >
```